server/service/src/invoice/inbound_shipment/add_from_master_list.rs

```diff
@@ -38,7 +38,7 @@ pub fn add_from_master_list(
         .connection
         .transaction_sync(|connection| {
             let invoice_row = validate(connection, &ctx.store_id, &input)?;
-            let new_invoice_line_rows = generate(connection, ctx, invoice_row, &input)?;
+            let new_invoice_line_rows = generate(ctx, invoice_row, &input)?;
```

Comment:
It's a good idea to add a github comment for a side quest like this, luckily I know the context here, but normally it's good to point it out otherwise the reviewer would be confused why this code has changed.
In general we try to keep side quests to a minimum, unless it's closely related to the main change.

Reply:
Ah yes good to know, I've added a note in the original comment